source/extensions/omni.isaac.lab/omni/isaac/lab/sim/converters/mesh_converter.py

```diff
@@ -111,6 +111,20 @@ def _convert_asset(self, cfg: MeshConverterCfg):
                     )
         # Delete the old Xform and make the new Xform the default prim
         stage.SetDefaultPrim(xform_prim)
+        # Apply default Xform rotation to mesh -> enable to set rotation and scale
+        omni.kit.commands.execute(
+            "CreateDefaultXformOnPrimCommand",
+            prim_path=xform_prim.GetPath(),
+            **{"stage": stage},
+        )
+        if cfg.rotation:
+            # Rotate mesh so that it is Z-up in the world
+            attr_rotate = xform_prim.GetAttribute("xformOp:orient")
+            assert attr_rotate.Set(Gf.Quatd(*cfg.rotation))
+        if cfg.scale:
+            # Scale mesh to meters
+            attr_scale = xform_prim.GetAttribute("xformOp:scale")
+            assert attr_scale.Set(Gf.Vec3d(*cfg.scale))
```

Comment:
Shouldn't this be done on the geom prim instead of the `xform` prim? Because later if you load it using the spawner config, the value on the form prim will be overwritten?

Reply:
Problem is that the geom prim does not have the scale and orient attribute. It there is a nice way to add them, then I can adopt it. In the previos version, this was also done by adding and Xform

Reply:
@Mayankm96 @pascal-roth resolved this, please double check